libs/SalesforceSDKCommon/SalesforceSDKCommon/Classes/Keychain/KeychainManager.swift

```diff
@@ -380,19 +383,15 @@ public class KeychainHelper: NSObject {
     /// Use this to relax or change the accessibility attribute for keychain items.
     /// - Parameter secAttrAccessible: Should be the accessibility attribute as defined by
     /// - Returns: KeychainResult
-    @objc public class func setAccessibleAttribute(_ secAttrAccessible: KeychainItemAccessibility) -> KeychainResult {
+    @objc @discardableResult
+    public class func setAccessibleAttribute(_ secAttrAccessible: KeychainItemAccessibility) -> KeychainResult {
        
+        accessibilityAttribute = secAttrAccessible.asCFString
         if accessibleAttributeMatches(secAttrAccessible) {
             return KeychainResult.init(data: Data(), status: errSecSuccess)
         }
         
         CachedWrapper.clearAllCaches()
-        let accessibleAttribute = secAttrAccessible.asCFString
```

Comment:
Why drop this?

Reply:
I moved it up before the `accessibleAttributeMatches(secAttrAccessible)` check so that even if it matches what's already there, we'll know that the app set the accessible attribute